postgresql-centos7-atomicapp/artifacts/kubernetes/postgresql-pod.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+id: postgresql
+spec:
+  containers:
+    - name: postgresql
+      image: centos/postgresql
```

Comment:
Could you have a look at the openshift postgres stuff at https://github.com/openshift/postgresql/blob/master/9.4/Dockerfile figure out what we can do with it or at least use it.